README.md

```diff
@@ -2,8 +2,8 @@
 
 [![Language](https://img.shields.io/badge/python-3.5%2B-blue.svg)](https://www.python.org/)
 [![License](https://img.shields.io/badge/license-GPL-yellow.svg)](https://github.com/aria-tools/ARIA-tools/blob/master/LICENSE)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/b69726c41ff9498882088afe57fe8391)](https://app.codacy.com/app/ehavazli/ARIA-tools?utm_source=github.com&utm_medium=referral&utm_content=ehavazli/ARIA-tools&utm_campaign=Badge_Grade_Settings)
-[![CircleCI](https://circleci.com/gh/ehavazli/ARIA-tools.svg?style=svg)](https://circleci.com/gh/ehavazli/ARIA-tools)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/ab9ba02d44c440fdac034cb144278b33)](https://www.codacy.com/app/ehavazli/ARIA-tools_2?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=aria-tools/ARIA-tools&amp;utm_campaign=Badge_Grade)
```

Comment:
The codacy badge still points to you. Is that correct?

Reply:
I think it is because I run it from my account. if you look at the last part of the line it says aria-tools/ARIA-tools

```
[![Codacy Badge](https://api.codacy.com/project/badge/Grade/ab9ba02d44c440fdac034cb144278b33)](https://www.codacy.com/app/ehavazli/ARIA-tools_2?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=aria-tools/ARIA-tools&amp;utm_campaign=Badge_Grade
```